2020/05/10 10:32:58 [error] 6#6: *3 FastCGI sent in stderr: "PHP message: 2020-05-10T10:32:58+00:00 [critical] Uncaught Exception: Enabling the fos_rest.param_fetcher_listener option when the Symfony Validator component is not installed is not supported. Try installing the symfony/validator package" while reading upstream, client: 172.19.0.1, server: localhost, request: "GET /todos HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 10:33:05 [error] 6#6: *3 FastCGI sent in stderr: "PHP message: 2020-05-10T10:33:05+00:00 [critical] Uncaught Exception: Enabling the fos_rest.param_fetcher_listener option when the Symfony Validator component is not installed is not supported. Try installing the symfony/validator package" while reading upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 13:59:36 [error] 6#6: *34 FastCGI sent in stderr: "PHP message: 2020-05-10T13:59:36+00:00 [critical] Uncaught Exception: Cannot autowire service "App\Domain\Controller\DeveloperController": argument "$serializer" of method "__construct()" references class "Symfony\Component\Serializer\Serializer" but no such service exists. Try changing the type-hint to one of its parents: interface "Symfony\Component\Serializer\SerializerInterface", interface "Symfony\Component\Serializer\Normalizer\NormalizerInterface", interface "Symfony\Component\Serializer\Normalizer\DenormalizerInterface", interface "Symfony\Component\Serializer\Encoder\EncoderInterface", or interface "Symfony\Component\Serializer\Encoder\DecoderInterface"" while reading upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 14:48:12 [error] 6#6: *95 FastCGI sent in stderr: "PHP message: 2020-05-10T14:48:12+00:00 [critical] Uncaught Exception: Invalid service "serializer.normalizer.object": method "Symfony\Component\Serializer\Normalizer\ObjectNormalizer::setCircularReferenceLimit()" does not exist" while reading upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 14:48:53 [error] 6#6: *95 FastCGI sent in stderr: "PHP message: 2020-05-10T14:48:53+00:00 [critical] Uncaught Exception: Invalid service "serializer.normalizer.object": method "Symfony\Component\Serializer\Normalizer\ObjectNormalizer::setCircularReferenceLimit()" does not exist" while reading upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 17:59:36 [error] 6#6: *146 FastCGI sent in stderr: "PHP message: 2020-05-10T17:59:36+00:00 [critical] Uncaught Exception: Could not open file or directory "/var/www/config/serialization"" while reading upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 18:01:30 [error] 6#6: *149 FastCGI sent in stderr: "PHP message: 2020-05-10T18:01:30+00:00 [critical] Uncaught Exception: Unrecognized option "max_depth" under "framework.serializer". Available options are "circular_reference_handler", "enable_annotations", "enabled", "mapping", "max_depth_handler", "name_converter"" while reading upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 18:02:13 [error] 6#6: *149 FastCGI sent in stderr: "PHP message: 2020-05-10T18:02:13+00:00 [critical] Uncaught Exception: The service "serializer" has a dependency on a non-existent service "1"" while reading upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 18:47:13 [error] 9#9: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 18:47:13 [error] 11#11: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /robots.txt HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 18:47:13 [error] 11#11: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost"
2020/05/10 19:21:55 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost:8000"
2020/05/10 19:21:56 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost:8000"
2020/05/10 19:21:58 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost:8000"
2020/05/10 19:22:00 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost:8000"
2020/05/10 19:22:01 [error] 6#6: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: localhost, request: "GET /api/developers HTTP/1.1", upstream: "fastcgi://172.19.0.5:9000", host: "localhost:8000"
